etna/records/models.py

```diff
@@ -91,20 +91,49 @@ def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
         Override this method on subclasses to add data that is relevant to a
         specific record type.
         """
+
+        def get_customdimension11(self):
```

Comment:
How would you feel about making these property methods and sticking with python's underscored naming as a rule? e.g:

```
@property
custom_dimension_11(self):
    pass
```

Reply:
Thanks @ababic Done this change.